frontend/13f-app/src/components/results.js

```diff
@@ -17,16 +18,16 @@ function Results () {
   }, [])
   return (
         <div>
-            <table>
+            <table id="companies">
               <tbody>
                 <tr>
                     <th>Company Name</th>
-                    <th>CIK_NO</th>
+                    <th>Number of Filings</th>
                 </tr>
                 {results.map(result => (
                     <tr key={result.cik_no}>
-                        <td>{result.company_name}</td>
-                        <td>{result.cik_no}</td>
+                        <td><Link to={`/company/${result.cik_no}`} id="linkstyle">{result.company_name}</Link></td>
```

Comment:
```suggestion
                        <td><Link to={`/company/${result.cik_no}`} className="link-style">{result.company_name}</Link></td>
```
[See here](https://github.com/eharley19/filing13F_searcher/pull/100/files#r707832578)

Reply:
You may also want to consider a more descriptive class name for this style, as you may end up adding different link styles for different pages in the future.
```css
.company-page-link-style
```
Or something along those lines.